### PR TITLE
fixed a bash scripting bug in start.sh

### PR DESCRIPTION
**Pull request by koehn**
that prevented horizon from ever starting.

### CODE REVIEW

File: contrib/docker/start.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ php artisan migrate --force
 php artisan update
 
 # Run a worker if it is set as embedded
-if [ $HORIZON_EMBED = true ]; then
+if [ "$HORIZON_EMBED" -eq "true" ]; then
```
Comment (dereckson) [inline contrib/docker/start.sh]: -eq is for integers, = is well the comparison operator to use for strings.

The real bug was the lack of "" (and that should be fixed).

**Review by dereckson — CHANGES_REQUESTED** [review]
Keep = operator, add quotes.

**Review by dansup — CHANGES_REQUESTED** [review]
Fix merge conflict

**Comment by koehn** [comment]
Changes merged.

**Review by dansup — APPROVED** [review]

**Pull request merged**